guides/languages.md

```diff
@@ -32,10 +32,9 @@ branch.
 - SPSS (legacy)
  
 ## Platform Management
-- Powershell (for DevOps / orchestration)
-- CLI
```

Comment:
Let me check in with whomever raised this before removing. If it's still in use (in anger) we should include it

Reply:
Main point for CLI was that it is not a language. It's a type of application.